__tests__/app.test.js

```diff
@@ -17,6 +17,27 @@ describe('GET /api/categories', () => {
         return request(app).get("/api/categories").expect(200);
     });
     test('an array of category objects, each of which should have the properties: "slug" & "description" ', () => {
-        return request(app).get("/api/categories").expect(200);
+        return request(app).get("/api/categories").expect(200)
+        .then(({body:categories})=>{
+            expect(typeof categories).toBe('object')
+            // console.log(categories)
+            categories.forEach((category)=>{
+                expect(Object.keys(category)).toHaveLength(2)
+                expect(category).toEqual(
+                    expect.objectContaining({
+                        slug: expect.any(String),
+                        description: expect.any(String)
+                    })
+                );
+            })
+        })
+    });
+
+describe('error handling', () => {
+    test('should respond with 404 server error if incomplete path recieved', () => {
+        // console.log('error handling')
+        return request(app).get("/api").expect(404)
```

Comment:
would be nice to test for the error message here too rather than just the error status 
(e.g {msg: "route not found"})